### PR TITLE
clear sessions revokes tokens

### PR DESCRIPTION
**Pull request by danielzeller-okta**
- Documents new query param on the clear sessions endpoint which revokes OAuth tokens

@royalchan-okta  @richardmateosian-okta 
@federations-okta

### CODE REVIEW

File: _source/_docs/api/resources/users.md [hunk 1]
```diff
@@ -2308,16 +2308,17 @@ Content-Type: application/json
 
 <span class="api-uri-template api-uri-delete"><span class="api-label">DELETE</span> /api/v1/users/*:uid*/sessions
 
-Removes all active identity provider sessions. This will force the user to authenticate on the next operation.
+Removes all active identity provider sessions. This will force the user to authenticate on the next operation. Optionally revokes OpenId Connect and OAuth tokens issued to the user.
```
Comment (royalchan-okta) [inline _source/_docs/api/resources/users.md]: we normally use `OpenID` in terms of capitalization right? Same for below.

File: _source/_docs/api/resources/users.md [hunk 2]
```diff
@@ -2308,16 +2308,17 @@ Content-Type: application/json
...
 #### Request Parameters
 {:.api .api-request .api-request-params}
 
-Parameter    | Description                                         | Param Type | DataType | Required | Default
------------- | --------------------------------------------------- | ---------- | -------- | -------- | -------
-uid          | `id` of user                                        | URL        | String   | TRUE     |
+Parameter    | Description                                                     | Param Type | DataType | Required | Default
+------------ | --------------------------------------------------------------- | ---------- | -------- | -------- | -------
+uid          | `id` of user                                                    | URL        | String   | TRUE     |
+oauthTokens  | Whether or not to revoke issued OpenId Connect and OAuth tokens | Query      | Boolean  | FALSE    | FALSE
```
Comment (cwu-okta) [inline _source/_docs/api/resources/users.md]: I would shorten this to something like
```Revoke issued OpenId Connect and OAuth tokens```

You'll notice when we have boolean variables this is how we typically describe them, ie see http://developer.okta.com/docs/api/resources/apps.html#add-application

File: _source/_docs/api/resources/users.md (hunk 1 above)
Comment (johngronberg-okta) [inline _source/_docs/api/resources/users.md]: This should specify which types of tokens are revoked. Just refresh and access tokens.

**Comment by danielzeller-okta** [comment]
@cwu-okta @royalchan-okta @johngronberg-okta I've made the changes you requested

**Comment by cwu-okta** [comment]
LGTM 🚢

File: _source/_docs/api/resources/users.md [hunk 3]
```diff
@@ -2308,16 +2308,17 @@ Content-Type: application/json
 
 <span class="api-uri-template api-uri-delete"><span class="api-label">DELETE</span> /api/v1/users/*:uid*/sessions
 
-Removes all active identity provider sessions. This will force the user to authenticate on the next operation.
+Removes all active identity provider sessions. This will force the user to authenticate on the next operation. Optionally revokes OpenID Connect and OAuth refresh and access tokens issued to the user.
```
Comment (richardmateosian-okta) [inline _source/_docs/api/resources/users.md]: Use present tense here: "this forces"

File: _source/_docs/api/resources/users.md [hunk 4]
```diff
@@ -2308,16 +2308,17 @@ Content-Type: application/json
...
 
 **Note:** This will not clear the sessions created for web sign-in or native applications.
```
Comment (richardmateosian-okta) [inline _source/_docs/api/resources/users.md]: Present tense again: "This does not clear"

File: _source/_docs/api/resources/users.md (hunk 3 above)
Comment (danielzeller-okta) [inline _source/_docs/api/resources/users.md]: @richardmateosian-okta I can make the change, but a counter-point: this new authentication will not occur immediately, but will occur at some unknown time in the future. Still want the change?

File: _source/_docs/api/resources/users.md (hunk 4 above)
Comment (danielzeller-okta) [inline _source/_docs/api/resources/users.md]: changed

File: _source/_docs/api/resources/users.md (hunk 3 above)
Comment (richardmateosian-okta) [inline _source/_docs/api/resources/users.md]: I think "authenticate on the next operation" covers the future part. I'd say "this forces."

Comment (danielzeller-okta) [inline _source/_docs/api/resources/users.md]: done

**Comment by mystiberry-okta** [comment]
Merged outside bacon because the associated work record already closed.
